gui/gui/src/TGSpeedo.cxx

```diff
@@ -387,6 +413,11 @@ void TGSpeedo::SetScaleValue(Float_t val)
    if (fValue > fPeakVal)
       fPeakVal = fValue;
 
+   fBuffer[fBufferCount % fBufferSize] = fValue;
+   ++fBufferCount;
+   if (fBufferCount >= fBufferSize)
+      fBufferCount = 0;
```

Comment:
This line is not needed any more, as the fBuffer[fBufferCount % fBufferSize]  would already take care of it?

Reply:
In principle yes, it's just to prevent integer overflow

Reply:
Ah ok, fair enough. (or changing to ulong64)